openedx/core/djangoapps/profile_images/tests/test_views.py

```diff
@@ -405,13 +385,79 @@ def test_remove_staff(self, mock_log):
     @patch('student.models.UserProfile.save')
     def test_remove_failure(self, user_profile_save, mock_log):
         """
-        Test that when upload validation fails, the proper HTTP response and
+        Test that when remove validation fails, the proper HTTP response and
         messages are returned.
         """
         user_profile_save.side_effect = [Exception(u"whoops"), None]
         with self.assertRaises(Exception):
-            self.client.post(self.url)
+            self.client.delete(self.url)
         self.check_images(True)  # thumbnails should remain intact.
         self.check_has_profile_image(True)
         self.assertFalse(mock_log.info.called)
         self.assert_no_events_were_emitted()
+
+
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Profile Image API is only supported in LMS')
```

Comment:
Yes, that's correct. There is a longer term plan to add profile images to Studio, so that course teams can use them. In fact, @marcotuts has a Hackathon proposal to do exactly this:

https://openedx.atlassian.net/wiki/display/OPEN/Hackathon+11%3A+Return+of+the+Con+Hackathon